.travis.yml

```diff
@@ -29,7 +29,6 @@ matrix:
     - env: PRODUCT_TESTS_SUITE=suite-1
     - env: PRODUCT_TESTS_SUITE=suite-2
     - env: PRODUCT_TESTS_SUITE=suite-3
-    - env: PRODUCT_TESTS_SUITE=suite-4
     - env: PRODUCT_TESTS_SUITE=suite-5
```

Comment:
should you update suite-5 to suite-4 and so on?

Reply:
i thought about that and decided not to.

Reply:
maybe we should change the suite name pattern
```
suite-1 
...
suite-4
non-generic-suite-1
...
```
?

Reply:
i prefer the current naming.